config/default.yml

```diff
@@ -24,6 +24,8 @@ public:
       authorizedOrigins: 'https://api.opensuse.org'
     pypi:
       baseUri: 'https://pypi.org'
+    snapcraft:
+      baseUri: 'https://api.snapcraft.io/v2/snaps/info'
```

Comment:
Does this actually need to be a setting?
AFAIK self-hosted snapcraft isn't a thing. Snapd is tied to https://snapcraft.io/
If this is something that needs to be configurable, can you link me to some docs about this
If it isn't, lets bin all this and hard-code the base url in `SnapcraftBase`

Reply:
You're right it is better to maintain in the base service.
I was wondering if someone would make tests using a local endpoint but in this case is sufficient to change the address inside the service base class!